README.md

```diff
@@ -32,6 +32,37 @@ The repository has the following permanent branches:
 
  * **apk** This branch contains the apks for the code in the develop branch. The apks are automatically updated when a commit is pushed to `develop` branch.
 
+
+##Run Server Locally To Test Android
+
+ * **Run server on Local machine methods.**
+	1. Run by local ip :-
+		 Connect your mobile and laptop on the same network. Type command ipconfig if you are using Linux or ipconfig if you are using windows. Modify the run.py by following line 
+
+		if __name__ == "__main__":
+		application.debug=True
+		application.run(host='<local-ip>',port=5000)
+	        <local-ip> be like 192.168.xx.xx
+	2. By using ngrok :-
```

Comment:
nice service, i had no idea about this 🤔

Reply:
You ca access that url form anywhere, then you didn't need to connect to that wifi. You require only internet to access that url.